bot.py

```diff
@@ -112,13 +124,119 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
 
+    #TODO: Purge all this shit
+    # elif "team_join" == response["type"]:
+    #     if NEWUSERGREETING:
+    #         new_user_message(response)
+    #     USER_LIST = sc.api_call(  # noqa: 841
+    #         "users.list",
+    #     )
+
+    log(response)   
+
+def log(response):
```

Comment:
What is it logging? Is it logging messages, emojis, or what? What in particular is being logged within those messages or emojis etc